### DISCUSSION TITLE
Not launching console at startup..

### DISCUSSION THREAD

**Discussion by EternalFury** [KlipperScreen/KlipperScreen#652] [Q&A]
I've installed klipper screen on my Voron printer(Voron 2.4, Octopus 1.1 board, RaspPi 3b+, Fluid, Klipper, Moonraker). 
I've reinstalled klipperscreen a dozen times. I even started from scratch and rebuilt the Fluid/klipper/moonraker image from scratch twice. I've went through all the trouble shooting from the Klipper site, I've followed Nero3d's video how to, and I just can NOT get the klipperscreen to boot into the console mode. My screen works, but I can't get the 3d printer console to start. I just see the normal pi txt as it starts up. 
All the video's I see of people setting Klipperscreen up, after they install the software, they reboot, and it just works on the screens. 
Mine just shows the raspi's startup screen.
![klipperscreen](https://user-images.githubusercontent.com/51927581/174509631-a79ea36b-71c8-4ff3-a8ad-37b3d2e37339.jpg)

Anyone with suggestions? And I'm sorry, but I'm very new to rasberry pi's and the associated software, so explain like I'm 5 please... I'm trying to learn.

**Comment by alfrix** [comment]
the logfiles are what tells you what is going on, they are needed to troubleshoot, please read: https://klipperscreen.readthedocs.io/en/latest/Troubleshooting/

**Reply by midmadn** [reply]
You need to login to your Pi with your username and password - then run sudo raspi-config. Set the option to auto login. Then save and exit.

**Comment by alfrix** [comment]
no log file was provided to troubleshoot the issue
